cogs/giftdrop.py

```diff
@@ -403,18 +404,25 @@ async def change_nickname_command(self, ctx: commands.Context, target: discord.M
         if not self.bot.db_available.is_set():
             return
 
+        async with self.bot.db.acquire() as conn:
+            if not check_is_in(conn, ctx.author.id):
```

Comment:
Only mods can run this command to change another user's nickname. So this should be checking the `target` not the author, and the error message reflect you're changing someone else's nickname.

Reply:
Should be dealt with now, nice catch